lib/resque/server/views/queues.erb

```diff
@@ -2,7 +2,7 @@
 
 <% if current_queue = params[:id] %>
 
-  <h1>Pending jobs on <span class='hl'><%= current_queue %></span></h1>
+  <h1>Pending jobs on <span class='hl'><%= h current_queue %></span></h1>
```

Comment:
`h` is only available in rails and resque does not depend on rails. This should do the trick:

```suggestion
  <h1>Pending jobs on <span class='hl'><%= ERB::Util.html_escape current_queue %></span></h1>
```

@brianvans does this look good to you?

Reply:
`h` is aliased as a helper over here:

https://github.com/brianvans/resque/blob/70f14b12d495ac0727cfe69fbfa2209fe2c42a31/lib/resque/server.rb#L28-L30

I noticed it was used in a few other views, so it made sense to use it here as well.

Reply:
Ah, nice! Didn't realize that.